services/github/github-commits-since.tester.js

```diff
@@ -33,6 +33,20 @@ t.create('Commits since by latest release (branch)')
     message: isMetric,
   })
 
```

Comment:
We have some schema validation helpers defined in a few helper modules within the `services` directory that help reduce the `@hapi/joi` boilerplate/repetition, including `withRegex` in `test-validators.js` that you could use here

Reply:
ok sure done, added another commit to refactor this in